pyveda/vedaset/stream/vedastream.py

```diff
@@ -62,13 +65,19 @@ def __init__(self, allocated, vset):
             self.exhausted = True
 
     def __len__(self):
-        return self.allocated
+        return len(self._vset._buf)
 
     def __iter__(self):
         return self
 
     def __getitem__(self, idx):
-        return [self.images[idx], self.labels[idx]]
+        try:
```

Comment:
this is NOT what we want here, we just want to raise indexerror like i mentioned in the commit for this